plugins/quickicon/eos310/eos310.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Quickicon.eos310
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+
+/**
+ * Joomla! end of support notification plugin
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class PlgQuickiconEos310 extends JPlugin
+{
+	/**
+	 * The EOS date for 3.10 to be updated at the stable release
+	 *
+	 * @var    string
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const EOS_DATE = '07.06.2024';
```

Comment:
If you write this in the same format as the php versions it will be more consistent and it works exactly the same
```suggestion
	const EOS_DATE = '2024-07-06';
```